docs/03-dev-howtos/01-ab-testing.md

```diff
@@ -295,7 +305,7 @@ It adds a rather large overhead to the cookie (mine is 2.5kb).
 
 # Write a Server-Side Test
 
-If you want to set up your AB test with scala instead of javascript, almost all the steps are different.
+If you want to set up your AB test to run away from the client ie. with scala, or in server-side rendered javascript, almost all the steps are different.
```

Comment:
I think the original sentence is a bit weird and we could probably remove it? The choice of implementing a server side or client side tests is not really related to a language preference.